syntax = "proto3";

package tvsc.pubsub;

message TestRequest {}

message TestResponse {
  string message = 1;
}

service TestService {
  rpc server_stream(TestRequest) returns (stream TestResponse);
}
